docs/azure/services/Microsoft_Purview_Information_Protection_Overview.md

```diff
@@ -0,0 +1,96 @@
+
+### Sensitivity Labels
+
+New and existing sensitivity labels can be found under the "Information Protection" tab on the compliance portal.
+
+![](../images/InformationProtection.png)
+
+The following actions can be achieved by labelling the data:
+- Encrypt emails, meeting invites, and documents to prevent unauthorized people from accessing this data
+- Mark created content when you use Office apps. This is done by adding watermarks, headers or footers to emails, meeting invites and documents that have a sensitivity label applied.
+- Protect content stored in containers such as sites and groups when you enable the feature to use sensitivity labels with Microsoft Teams, Microsoft 365 groups, and SharePoint sites.
+- Apply the label automatically to all files and emails, or recommend a label.
+- Set the default sharing link type for SharePoint sites and individual documents.
+
+When configuring a sensitivity label, the following options are presented to a user:
+
+#### Scope
+
+Defines what data type labels will be applied to. As of now, there are three options available for selection: 
+- Items (includes emails and files)
+- Groups & Sites (includes Teams, O365 groups as well as SharePoint sites)
+- Schematised data assets (includes files and schematised data access in Data Map)
+
+#### Protection level
+
+Defines what action(s) the label will do. There are two options: encryption (controls who and what level of access the specified user has) and mark items (adds watermark to files). By opting for encryption, the user is presented with a encryption configuration menu in which the user can configure the following settings:
+- Assign permissions now or let users decide
+- User access expiration
+- Allow or disallow offline access - users accessing the file need to be connected to the internet
+- Assign permissions to specific groups - for example, all users in the HR group can edit and view the files, whilst every other internal user can only view file.
+
+#### Auto-labelling for files and emails
+
+This option provides a way to configure how the data will be discovered. As mentioned above, it's possible to discover it manually, using sensitive info type and trainable classifiers. If opted for SIT, patterns will be available for review in this configuration menu. Once a user saves the document, the label will automatically be applied.
+ 
+#### Groups & Sites 
+
+If the user opted for "Groups & Sites" under the scope configuration menu, this configuration will allow the user to define the protection setting for groups and data. This includes the level of access that internal and external users will have for O365 and Microsoft Teams groups, as well as sharing policies of SharePoint sites with external users.
+
+Once a label is configured with the above policies, it must be published. Labels can be published to all or selected users and groups. Furthermore, the following settings are available prior to publication:
```

Comment:
Can we clarify that these are default settings that could be configured prior to publishing a label? Thus enforcing some extra security controls.